packages/roosterjs-content-model-plugins/lib/edit/inputSteps/handleEnterOnParagraph.ts

```diff
@@ -11,6 +11,7 @@ export const handleEnterOnParagraph: DeleteSelectionStep = context => {
 
     if (context.deleteResult == 'notDeleted' && paraIndex >= 0) {
         const newPara = splitParagraph(context.insertPoint);
+        delete newPara.format.id;
```

Comment:
1. How about move this into function `splitParagraph`? I can't think about any scenario that we split paragraph and want the ID to be duplicated
2. Instead of explicitly delete ID here, suggest we just whitelist all format that we want to copy, then remove everything else.